### PR TITLE
Update UserInputService.yaml

### PR DESCRIPTION
**Pull request by IgnisRBX**
## Checks

By submitting your pull request for review, you agree to the following:

- [ ] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [ ] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [ ] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

**Comment by krashlife** [comment]
Good

File: content/en-us/reference/engine/classes/UserInputService.yaml
[the remark this answers is not in the document]
Comment (io1ismz) [reply content/en-us/reference/engine/classes/UserInputService.yaml:1]: ### _****_

[the remark this answers is not in the document]
Comment (Hahah1355) [reply content/en-us/reference/engine/classes/UserInputService.yaml:1]: Ok

**Comment by tyryfhrhgtrht** [comment]
Big the map

**Comment by tyryfhrhgtrht** [comment]
****

**Pull request closed without merging**